src/containers/pageDataLoadingAPI.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Export loadData calls from ducks modules of different containers
+ */
+import { setInitialValues as CheckoutPageInitialValues } from './CheckoutPage/CheckoutPage.duck';
+import { loadData as ContactDetailsPageLoader } from './ContactDetailsPage/ContactDetailsPage.duck';
+import { loadData as EditListingPageLoader } from './EditListingPage/EditListingPage.duck';
+import { loadData as EmailVerificationPageLoader } from './EmailVerificationPage/EmailVerificationPage.duck';
+import { loadData as InboxPageLoader } from './InboxPage/InboxPage.duck';
+import { loadData as ListingPageLoader } from './ListingPage/ListingPage.duck';
+import { loadData as ManageListingsPageLoader } from './ManageListingsPage/ManageListingsPage.duck';
+import { loadData as PaymentMethodsPageLoader } from './PaymentMethodsPage/PaymentMethodsPage.duck';
+import { loadData as ProfilePageLoader } from './ProfilePage/ProfilePage.duck';
+import { loadData as SearchPageLoader } from './SearchPage/SearchPage.duck';
+import { loadData as StripePayoutPageLoader } from './StripePayoutPage/StripePayoutPage.duck';
+import {
+  loadData as TransactionPageLoader,
+  setInitialValues as TransactionPageInitialValues,
+} from './TransactionPage/TransactionPage.duck';
+
+const pageDataLoadingAPI = () => {
```

Comment:
Why is this wrapped in a function that is already called in this file?

Reply:
@kpuputti actually, there's no good reason. 

At some point, I was planning to test how module import behaves in different export scenarios (execute function, IIFE, instance, etc.) Just to understand better the issue we originally faced with routeConfiguration (when it was not wrapped with a function that was called again where ever it is needed).

Anyway, I didn't go forward with that plan. So, I'll change this so that function is exported and it is called in routeConfiguration.js.

Although, we should probably explore class/instance setup at some point. Twitter's [reducer registry](http://nicolasgallagher.com/redux-modules-and-code-splitting/) approach is quite interesting. They export registry instance from reducerRegistry.js file and it's imported in all the duck-files they use. Then the duck's reducer is registered to that registry at the time that the duck file is bundled-in - and finally, the store asks all the reducers from the instance when the registry emits "change" events. Quite an interesting setup for code-splitting.